book/forms.rst

```diff
@@ -77,8 +77,8 @@ from inside a controller::
     // src/AppBundle/Controller/DefaultController.php
     namespace AppBundle\Controller;
 
-    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
     use AppBundle\Entity\Task;
+    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
```

Comment:
We don't separate use statements in several blocks in the code.

And we usually keep them sorted in new code, but we don't sort them in existing code when refactoring if there are not sorted properly to avoid useless conflicts when merging branches